README.md

```diff
@@ -87,12 +87,58 @@ jsep.removeIdentifierChar('@');
 
 ### Plugins
 JSEP supports defining custom hooks for extending or modifying the expression parsing.
-All hooks are bound to the jsep instance and called with a single argument and return void.
-The 'this' context provide access to the internal parsing methods of jsep
+Plugins are registered by calling `jsep.plugins.register(the_plugin)`.
```

Comment:
How do authors that wish to use plugins find what `the_plugin` identifier to use? To us it may be obvious, but to a beginner reading the docs, it is very opaque. 

Also, I think we should recommend the aggregate way of registering plugins, i.e. you import all the plugins, then you register them all.